joomla/joomla-cms
# [4.0] Remove checkin, checkout methods from ProfileModel

Pull Request for Issue # .

### Summary of Changes
Table  #__users does not have **checked_out** or **checked_out_time**, so the checkin, checkout method in ProfileModel are useless and could be removed

### Testing Instructions
1. Create a menu item to link to Edit User Profile menu option
2. Apply patch
3. Login to any account, access to that menu item to edit profile data
4. Press Save button, make sure the profile data is still being saved properly
